test/specs/views/Card/Card-test.js

```diff
@@ -0,0 +1,53 @@
+import faker from 'faker'
+import React from 'react'
+
+import * as common from 'test/specs/commonTests'
+import { sandbox } from 'test/utils'
+import Card from 'src/views/Card/Card'
+import CardContent from 'src/views/Card/CardContent'
+import CardDescription from 'src/views/Card/CardDescription'
+import CardGroup from 'src/views/Card/CardGroup'
+import CardHeader from 'src/views/Card/CardHeader'
+import CardMeta from 'src/views/Card/CardMeta'
+
+describe('Card', () => {
+  common.isConformant(Card)
+  common.hasUIClassName(Card)
+  common.hasSubComponents(Card, [CardContent, CardDescription, CardGroup, CardHeader, CardMeta])
+  common.propKeyOnlyToClassName(Card, 'centered')
+  common.propKeyOnlyToClassName(Card, 'fluid')
+  common.propKeyOnlyToClassName(Card, 'raised')
+  common.rendersChildren(Card)
+
+  describe('renders different elements', () => {
```

Comment:
Prefer we reorganize these describe/it blocks a bit to be consistent with other components, reasoning below:

```jsx
  it('renders a <div> by default', () => {
    ...
  })

  describe('href', () => {
    it('renders an <a> with an href attr', () => {
      ...
    })
  })

  describe('onClick', () => {
    it('can be omitted', () => {
      ...
    })

    it('renders <a>', () => {
      ...
    })

    it('is called with (event) on click', () => {
      ...
    })
  })
```

I split some of the `it()`s into multiple individual assertions.  Just like writing single purpose functions, targeted test assertions are preferred.  It makes them stronger and easier to maintain as a single test will only fail for a single specific reason.

I moved the default div tagName assertion to the root of the `describe('Card')`.  This is because that assertion is asserting something about the card itself.  It will help with the console output and also reads nicely.  I find it helpful to read the `describe()` strings and `it()` in succession to help figure out how to split tests, since that is how the test runner will also print their failures.  Here's how some of my changes will print:

```diff
-Card renders different elements <div> by default
+Card renders a <div> by default

-Card renders different elements <a> with `href` prop
+Card href prop renders an <a> with an href attr
```